src/prog_models/sim_result.py

```diff
@@ -32,7 +36,56 @@ def __init__(self, times : list = None, data : list = None, _copy = True):
             else:
                 self.data = data
 
-    def __eq__(self, other : "SimResult") -> bool:
+    def __getitem__(self, item):
+        """
+        created for deprecation warning. [] continues to be handled by parent
+        """
+        warn(
+            '[] will be deprecated after version 1.5 of ProgPy. Users may use pandas Dataframe \'iloc\'  to access '
+            'data by element.',
+            DeprecationWarning, stacklevel=2)
+        return super().__getitem__(item)
+
+    def iloc(self, item):
```

Comment:
We want iloc to work like dataframes to ease transition. That means we need it to work with square brackets. (E.g., result.iloc[10]). 

Let me know if you need some help with this, I have an idea of how to do it